version-specific-2.9/ember-internals.js

```diff
@@ -0,0 +1 @@
+export function initialize() {}
```

Comment:
This is unfortunate in that it results in an entire noop module. :/ I'm tempted to propose removing [this line](https://github.com/ember-a11y/ember-a11y/pull/48/files#diff-404e1e58743da7f3a7d8f13cc900e6cbR4) at build time to avoid this.